### ISSUE TITLE
Add animated GIF showing different style possibilities

### ISSUE THREAD

**Issue by coastalwhite** [coastalwhite/lemurs#86]
It would be nice to replace the current banner with an animated GIF that shows different styles so that we can illustrate configurability.

**Comment by amsyarzero** [comment]
Sorry to bring this up, but by banner, do you mean the **Screenshot** part in README.md?

**Comment by coastalwhite** [comment]
Yes, exactly. This could be done in a similar way as was done for [wavedrom-rs](https://github.com/coastalwhite/wavedrom-rs).

Your help would be very much appreciated.
